### ISSUE TITLE
Playback and recording meters: peak level and clipping indication are not accessible to users of screen readers.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#11650]
### Steps to reproduce

1. Run a screen reader
2. Try to find out the peak level and whether clipping has occurred for a playback/recording meter

### Expected behavior

You can read this info.

### Actual behavior

You can't read this info.

### Audacity Version

Other

### Operating system

Windows

### Additional context

Tested using commit c5a20ac

On Windows, this is a regression wrt Audacity 3.
